app/code/Magento/Wishlist/Block/Share/Email/Items.php

```diff
@@ -11,17 +11,57 @@
  */
 namespace Magento\Wishlist\Block\Share\Email;
 
+use Magento\Catalog\Model\Product;
+use Magento\Catalog\Model\Product\Configuration\Item\ItemResolverInterface;
+use Magento\Catalog\Model\Product\Image\UrlBuilder;
+use Magento\Framework\View\ConfigInterface;
+use Magento\Wishlist\Model\Item;
+
 /**
  * @api
  * @since 100.0.2
  */
 class Items extends \Magento\Wishlist\Block\AbstractBlock
 {
+    /** @var ItemResolverInterface */
+    private $itemResolver;
+
     /**
      * @var string
      */
     protected $_template = 'Magento_Wishlist::email/items.phtml';
 
+    /**
+     * @param \Magento\Catalog\Block\Product\Context $context
+     * @param \Magento\Framework\App\Http\Context $httpContext
+     * @param ItemResolverInterface $itemResolver
+     * @param array $data
+     * @param ConfigInterface|null $config
+     * @param UrlBuilder|null $urlBuilder
+     */
+    public function __construct(
+        \Magento\Catalog\Block\Product\Context $context,
+        \Magento\Framework\App\Http\Context $httpContext,
+        ItemResolverInterface $itemResolver,
+        array $data = [],
+        ConfigInterface $config = null,
+        UrlBuilder $urlBuilder = null
+    ) {
+        $this->itemResolver = $itemResolver;
+        parent::__construct($context, $httpContext, $data, $config, $urlBuilder);
```

Comment:
Please make sure that you call the parent construct first.
```suggestion
        parent::__construct($context, $httpContext, $data, $config, $urlBuilder);
        $this->itemResolver = $itemResolver;
```